Sources/SwiftKafka/KafkaConsumer.swift

```diff
@@ -55,32 +85,26 @@ public struct ConsumerMessagesAsyncSequence: AsyncSequence {
     }
 }
 
+// MARK: - KafkaConsumer
+
 /// Receive messages from the Kafka cluster.
 public final class KafkaConsumer {
+    typealias Producer = NIOAsyncSequenceProducer<
+        Result<KafkaConsumerMessage, KafkaError>,
+        NoBackPressure,
+        ShutdownOnTerminate
+    >
+    /// Time between two consecutive polls.
+    private var pollInterval: Duration
```

Comment:
Why is this not part of the `KafkaConsumerConfiguration`?

Reply:
Wanted to do that in a separate PR as this also affects `KafkaProducer`, but yeah I can do it in this one already

Reply:
Separate PR is fine

Reply:
Too late now 😅